MODULE.bazel

```
@@ -26,3 +26,4 @@ bazel_dep(name = "boost.system", version = "1.83.0.bzl.2")
 bazel_dep(name = "boost.throw_exception", version = "1.83.0.bzl.1")
 bazel_dep(name = "boost.type_traits", version = "1.83.0.bzl.1")
 bazel_dep(name = "boost.utility", version = "1.83.0.bzl.1")
+bazel_dep(name = "openssl", version = "3.3.1.bcr.1")
```

```suggestion
bazel_dep(name = "openssl", version = "3.3.1.bcr.2")
```

We just released Windows support - hopefully this change should make things work! (Huge thanks to @andrewkatson for doing all of the hard work here!)